packages/data-context/src/actions/AuthActions.ts

```diff
@@ -48,6 +48,8 @@ export class AuthActions {
       this.ctx.update((coreData) => {
         coreData.authState = authState
       })
+
+      this.ctx.emitter.authChange()
```

Comment:
This actually fires before the login officially occurs. Wondering if we need to do it when the login officially happens:

https://github.com/cypress-io/cypress/blob/10.0-release/packages/server/lib/makeDataContext.ts#L80-L85

Reply:
Nice find - going to get a better idea of the flow here